### PR TITLE
Refactor styling for slider.

### PR DESCRIPTION
**Pull request by bierik**
Fixes https://github.com/4teamwork/bern.web/issues/175

Depends on https://github.com/OneGov/plonetheme.onegovbear/pull/14

### CODE REVIEW

File: ftw/slider/browser/resources/ftwtheming.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+$slider-button-bg-color: $gray-dark !default;
+$slider-button-text-color: $gray-lighter !default;
+
+@include declare_variables(
+  slider-button-size,
+  slider-button-gutter,
+  slider-text-bg-color,
+  slider-text-transparency,
+  slider-text-title-font-size,
+  slider-texttitle-font-color,
```
Comment (jone) [inline ftw/slider/browser/resources/ftwtheming.scss]: :question:

File: ftw/slider/profiles/default/cssregistry.xml [hunk 2]
```diff
@@ -2,7 +2,9 @@
 <object name="portal_css">
   <stylesheet id="++resource++ftw.slider/slider.css"
               enabled="True"
+              expression="not:request/HTTP_X_FTW_THEMING|nothing"
```
Comment (jone) [inline ftw/slider/profiles/default/cssregistry.xml]: This requires an upgrade step :hushed:

**Comment by bierik** [comment]
@jone updated

File: ftw/slider/upgrades/configure.zcml [hunk 3]
```diff
@@ -52,4 +52,14 @@
         profile="ftw.slider:default"
         />
 
+    <!-- 1114 -> 1115 -->
+    <upgrade-step:importProfile
+        title="Activate ftw.theming."
```
Comment (jone) [inline ftw/slider/upgrades/configure.zcml]: This does not activate anything. I'd name it
```Disable standard CSS with ftw.theming in favor of sass file```

**Comment by bierik** [comment]
@jone thanks for the hint... updated

**Pull request merged**